release_notes/Release_Notes.txt

```diff
@@ -5,10 +5,10 @@ bufr-dump v1.1.0        -> released to NCO on Aug 17,2022
 
 ---------------------------------------------------
 List of all code/scripts modified with this release
-  CMakeLists.txt
-  README.md
-M VERSION
-  build.sh
+M CMakeLists.txt
+M README.md
+D VERSION
+D build.sh
```

Comment:
Technically, these were moved, not deleted.  But Ok.

Reply:
We don't have a letter for it, so I used D(delete) and A (added) .But I agree